chirp/drivers/ft450d.py

```diff
@@ -42,30 +50,65 @@
 T_STEPS.remove(125.0)
 T_STEPS.remove(200.0)
 
-@directory.register
-class FT450DRadio(yaesu_clone.YaesuCloneModeRadio):
-    """Yaesu FT-450D"""
+
+def dbg_dump(str1, kn=32, cptn="DBG", ax=False, lgx=1):
+    """ Debugging tool to print up to kn hex values of list stz """
+    # ax: flag to append ascii, lgx: Log code (bits) 1=Warning, 2=Debug
+    sx = ""
+    styp = isinstance(str1, bytes)
+    kx = len(str1)
+    if kx > kn:
+        kx = kn
+    for ix in range(kx):
+        if not styp:
+            sx += "%02x " % ord(str1[ix])
+        else:
+            sx += "%02x " % str1[ix]
+    if ax:      # append as ascii
+        sx += " ["
+        for ix in range(kx):
+            if not styp:
+                valx = ord(str1[ix])
+                chrx = str1[ix]
+            else:
+                valx = str1[ix]
+                chrx = chr(str1[ix])
+            if valx > 31 and valx < 127:
+                sx += "%s" % chrx
+            else:
+                sx += "."
+        sx += "]"
+    if lgx & 1:
+        LOG.warning("%s: %s" % (cptn, sx))
+    if lgx & 2:
+        LOG.debug("%s: %s" % (cptn, sx))
+    return

```

Comment:
Is this necessary? Just using the existing hexprint function with a slice should be fine:

```
print(util.hexprint(buffer[:32]))
```